index.html

```diff
@@ -19,27 +19,27 @@ <h2 class="page-header text-center">
 <section ng-show="repoData" id="repositories" class="flex-container">
     <div ng-repeat="repo in repoData | filter:searchText  | orderBy:predicate:reverse" class="flex-item {{repo.name}} {{repo.owner.login}} {{repo.primaryLanguage.name}}">
         <h4 class="text-center">
-        <a href="{{ repo.url }}"> {{ repo.name }} </a>
-        <small><a href="https://github.com/{{ repo.owner.login }}"> {{ repo.owner.login }} </a></small>
+        <a href="{{ repo.url }}" alt="View Project on GitHub" title="View Project on GitHub"> {{ repo.name }} </a>
+        <small><a href="https://github.com/{{ repo.owner.login }}" alt="View Owner on GitHub" title="View Owner on GitHub"> {{ repo.owner.login }} </a></small>
         <small>{{ repo.primaryLanguage.name }}</small>
```

Comment:
Should we add a mouseover for the language? e.g. "Primary Project Language" or similar?

Reply:
Added